kratos/python_scripts/assign_mpcs_to_neighbours_process.py

```diff
@@ -0,0 +1,178 @@
+import KratosMultiphysics as KM
+import numpy as np
+
+def Factory(settings, model):
+    if not isinstance(settings, KM.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return AssignMPCsToNeighboursProcess(model, settings["Parameters"])
+
+## All the processes python should be derived from "Process"
+class AssignMPCsToNeighboursProcess(KM.Process):
+    """The process facilitates the discovery of neighboring nodes in a 
+    master model part within a designated radius for each node in the 
+    slave model part. Following this, it establishes a multipoint constraint (MPC) 
+    and calculates its weights using a spatial function that employs radial basis functions.
+
+    Public member variables:
+    model -- the container of the different model parts.
+    settings -- Kratos parameters containing process settings.
+    """
+
+    def __init__(self, model, settings):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        Model -- the container of the different model parts.
+        settings -- Kratos parameters containing process settings.
+        """
+        KM.Process.__init__(self) # calling the baseclass constructor
+
+        default_settings = KM.Parameters("""{
+            "computing_model_part_name": "",
```

Comment:
`model_part_name` to be consistent with the rest of processes. Also note that not all the solvers in Kratos require a computing model part.